apps/ledger-live-desktop/tests/page/modal/send.modal.ts

```diff
@@ -47,6 +60,20 @@ export class SendModal extends Modal {
     }
   }
 
+  @step("Craft NFT tx")
+  async craftNFTTx(tx: NFTTransaction) {
+    if (tx.accountToCredit.ensName) {
+      await this.fillRecipient(tx.accountToCredit.ensName);
+    } else {
+      await this.fillRecipient(tx.accountToCredit.address);
+    }
```

Comment:
Could be summarise with
`await this.fillRecipient(tx.accountToCredit.ensName || tx.accountToCredit.address);`